packages/format-library/src/time/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { gmdate, strtotime } from 'locutus/php/datetime';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { toggleFormat } from '@wordpress/rich-text';
+import {
+	RichTextToolbarButton,
+	RichTextShortcut,
+} from '@wordpress/block-editor';
+import { postDate as icon } from '@wordpress/icons';
+
+const name = 'core/time';
+const title = __( 'Time' );
+
+export const time = {
+	name,
+	title,
+	tagName: 'time',
+	className: null,
+	attributes: {
+		datetime: 'datetime',
+	},
+	edit( { isActive, value, onChange, onFocus } ) {
+		function onClick() {
+			const dateDescription = value.text
+				.slice( value.start, value.end )
+				.trim();
+
+			// Exit early if no selection or format is already active
+			if ( ! dateDescription || isActive ) {
+				onChange(
+					toggleFormat( value, {
+						type: name,
+					} )
+				);
+				return;
+			}
+
+			// Clean up the date string
+			const dateCleaned = dateDescription
+				.replace( 'at ', '' ) // Remove "at"
+				.replace( 'UTC', 'GMT' ); // Replace "UTC" with "GMT"
+
+			// Parse the date string using strtotime
+			const timestamp = strtotime( dateCleaned );
```

Comment:
I think we already use `moment` in other packages, can we use that?

Reply:
Umm could you provide some context about `moment`?

Reply:
I think what is being suggested here is to use [the `@wordpress/date` package](https://github.com/WordPress/gutenberg/blob/trunk/packages/date/README.md), which imports `moment` internally, instead of the `locutus` library. That package has various utility functions for working with dates. If we use those functions, we should not need to add the `locutus` library as a dependency.